### DISCUSSION TITLE
Use N8N.io instead of Zapier at Org AND consider adding it to Omnibus distribution

### DISCUSSION THREAD

**Discussion by aaronrothschild** [mattermost/community#10] [Practical Ideas]
N8N.io is an open source workflow system that helps integrate with external systems rather than writing code to connect them. 

N8N could host workflows for different business users who need to integrate Mattermost with their internal systems but don't know how to code.  

For future, we could add a better "connector/Node" for Mattermost that uses the App Framework to connect and make interactive apps for end users using low-code concepts.

**Comment by jprusch** [comment]
Great idea! Having a connector for Mattermost (or even Focalboard?) would be awsome. Should be able to trigger n8n workflows via slash commands or the new app integration & also able to retrieve and route messages with possible actionable buttons to dedicated channels.
